src/main/java/com/google/graphgeckos/dashboard/datatypes/RevisionStatus.java

```diff
@@ -0,0 +1,24 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.graphgeckos.dashboard.datatypes;
+
+/**
+ * Represents global revision statuses.
+ */
+public enum RevisionStatus {
+  failed,
```

Comment:
turn those cases into uppercase, like in https://github.com/googleinterns/step240-2020/blob/master/src/main/java/com/google/graphgeckos/dashboard/datatypes/BuilderStatus.java

Reply:
This requires further rework from the front-end part. I think it would be best to leave them as they are now, as per @samuelsogbesan 's request, and these will be changed on both the front and back end parts in a future PR

Reply:
ok. Please add a TODO here

Reply:
done